cloudconnexa/resource_settings.go

```diff
@@ -51,7 +51,7 @@ func resourceSettings() *schema.Resource {
 			"connect_auth": {
 				Type:         schema.TypeString,
 				Optional:     true,
-				ValidateFunc: validation.StringInSlice([]string{"NO_AUTH", "ON_PRIOR_AUTH", "EVERY_TIME"}, false),
+				ValidateFunc: validation.StringInSlice([]string{"NO_AUTH", "ON_PRIOR_AUTH", "EVERY_TIME", "SAML"}, false),
```

Comment:
There is no SAML connect_auth type, this will not work

Reply:
That's an interesting point, i'll look into it, thanks @vladhss